views/overlays/umsk/game16by9.hbs

```diff
@@ -25,6 +25,28 @@
         document.getElementById('rplayer').innerText = data.rplayer || "";
         document.getElementById('lscore').innerText = data.lscore || "";
         document.getElementById('rscore').innerText = data.rscore || "";
+
+        if(data.lCharacter) {
+          var leftIcon = document.createElement('img');
+          leftIcon.id = 'leftCharacter';
+          leftIcon.className = data.lCharacter;
+          leftIcon.style.zoom = '200%';
+          //src is required to remove the dumb border
+          leftIcon.src = 'http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif';
+          document.getElementById('lplayer').appendChild(leftIcon);
```

Comment:
Sweet, I'll merge after you do your thing.

On Fri, Jan 9, 2015 at 9:41 AM, Cameron Crockrom <notifications@github.com>
wrote:

> In views/overlays/umsk/game16by9.hbs
> <https://github.com/dguenther/web-overlay/pull/29#discussion-diff-22722054>
> :
>
> > @@ -25,6 +25,28 @@
> >          document.getElementById('rplayer').innerText = data.rplayer || "";
> >          document.getElementById('lscore').innerText = data.lscore || "";
> >          document.getElementById('rscore').innerText = data.rscore || "";
> > +
> > +        if(data.lCharacter) {
> > +          var leftIcon = document.createElement('img');
> > +          leftIcon.id = 'leftCharacter';
> > +          leftIcon.className = data.lCharacter;
> > +          leftIcon.style.zoom = '200%';
> > +          //src is required to remove the dumb border
> > +          leftIcon.src = 'http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif';
> > +          document.getElementById('lplayer').appendChild(leftIcon);
>
> This was what was happening.
> https://stackoverflow.com/questions/17851631/jquery-remove-the-innertext-but-preserve-the-html
>
> I'll get a fix soon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dguenther/web-overlay/pull/29/files#r22722054>.
>